scripts/accumulator-server.py

```diff
@@ -44,8 +44,15 @@
 import atexit
 import string
 import signal
+from OpenSSL import SSL
 
-
+# Set up TLS
+dir = os.path.dirname(argv[0])
+if dir == '':
+    dir = os.curdir
+context = SSL.Context(SSL.SSLv23_METHOD)
+context.use_certificate_file(os.path.join(dir, 'accumulator.crt'))
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/commit/90c8c15aa6200d9417a0e443ef2bd6c62b496cbc